### PR TITLE
V1.1.0 new reqs

### PR DESCRIPTION
**Pull request by tetron**
New v1.1 process requirements.

### CODE REVIEW

**Comment by tetron** [comment]
Based on #660 which needs to merge first.

**Comment by tetron** [comment]
TimeLimit, WorkReuse and InternetAccess need to be implemented in cwltool.  BatchQueue is a no-op in cwltool.  Semantics of Workspace in cwltool is debatable, could be used to set --tmp-outdir-prefix and --outdir.

**Comment by tetron** [comment]
@LourensVeen interested in your feedback: https://github.com/common-workflow-language/common-workflow-language/pull/661/commits/bc4971ac282ed33ba6f5a866d50867048d0c3119

File: v1.1.0-dev1/CommandLineTool.yml [hunk 1]
```diff
@@ -990,3 +990,139 @@ $graph:
...
+    particular tool or step.
+  mapSubject: enableReuse
+  fields:
+    - name: class
+      type: string
+      doc: "Always 'WorkReuse'"
+      jsonldPredicate:
+        "_id": "@type"
+        "_type": "@vocab"
+    - name: enableReuse
```
Comment (mr-c) [inline v1.1.0-dev1/CommandLineTool.yml]: What's the default value?

File: v1.1.0-dev1/CommandLineTool.yml [hunk 2]
```diff
@@ -990,3 +990,139 @@ $graph:
...
+      doc: "Always 'Workspace'"
+      jsonldPredicate:
+        "_id": "@type"
+        "_type": "@vocab"
+    - name: workspace
+      type: ["null", string, Expression]
+      jsonldPredicate:
+        "_type": "@id"
+      doc: |
+        URI for the.
```
Comment (mr-c) [inline v1.1.0-dev1/CommandLineTool.yml]: ?

**Comment by tetron** [comment]
Spiked `Workspace` and `BatchQueue` based on feedback.

cwltool implementations for TimeLimit, WorkReuse and NetworkAccess PR:

https://github.com/common-workflow-language/cwltool/pull/718

**Pull request merged**